schema/fields_test.go

```diff
@@ -101,4 +101,59 @@ func TestFieldBuilder_Build(t *testing.T) {
 		_, err := builder.Build()
 		require.Error(t, err)
 	})
+
+	t.Run("test programming language keywords as field name", func(t *testing.T) {
+		builder1 := &FieldBuilder{FieldName: "abstract", Type: "string"}
+		_, err1 := builder1.Build()
+		require.Equal(t, err1, api.Errorf(api.Code_INVALID_ARGUMENT, "Invalid collection field name, It contains language keyword for fieldName = 'abstract'"))
```

Comment:
sure updated it. I personally like to duplicate messages in test so that if we change message in future with the intention to change it at only one place and if it is being used at more than one place, our test will give us a light punch.

Reply:
Thanks @rbarabas this is much concise.